SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -1101,6 +1101,24 @@ class CSolver {
                                  unsigned short val_marker,
                                  bool val_inlet_surface) { }
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] conv_numerics - Description of the numerical method.
+   * \param[in] visc_numerics - Description of the numerical method.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] val_marker - Surface marker where the boundary condition is applied.
+   * \param[in] val_inlet_surface - Boolean for whether val_marker is an inlet
+   */
+  inline virtual void BC_ActDisk_VariableLoad(CGeometry *geometry,
+                                              CSolver **solver_container,
```

Comment:
This function does not need to be virtual, since it is only used in CEulerSolver, by CEulerSolver

Reply:
Did it, it is not anymore virtual.